src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -3093,16 +3082,129 @@ emit_vector_2_3_4 (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *f
 
 		return min;
 	}
-	case SN_Conjugate:
-	case SN_Distance:
-	case SN_DistanceSquared:
+	case SN_Distance: 
+	case SN_DistanceSquared: {
+#if defined(TARGET_ARM64)
```

Comment:
Add `return NULL` for unsupported architectures for all the API that you intrinsified in this PR.